### PR TITLE
New Readme: Setup And Git Commands

### PR DESCRIPTION
**Pull request by MichaelCherry**
Addition of Setup Steps and Git Commands in a readme file

### CODE REVIEW

File: rm_setup_and_git_commands.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+This file is to document environmental setup and git commands
+
+## Setup Dev Environment
+
+### Install needed applications
+
+Ensure that you have docker and docker-compose installed on your machine, as well as the rest of the Python stack.
+ * GitBash
+ * Python 3.6.7
```
Comment (caleybrock) [inline rm_setup_and_git_commands.md]: Should we also include node here?

**Comment by caleybrock** [comment]
In general, the files in this project are `camelCase` or `use-dashes-not-underscores`. Thoughts on using dashes on this one too, to be consistent?

File: rm_setup_and_git_commands.md (hunk 1 above)
Comment (MichaelCherry) [reply rm_setup_and_git_commands.md]: I think node is technically pulled in by Docker: we have 9.7 on our machines, but are using 10.x in Docker.

**Comment by MichaelCherry** [comment]
@caleybrock Replied to comment, changed filename to use -x rather than camel case.

**Pull request merged**